lerna.json

```diff
@@ -1,10 +1,10 @@
 {
-  "lerna": "2.4.0",
```

Comment:
This is redundant information now, they removed this from lerna.json (see https://github.com/lerna/lerna/pull/1225).